### DISCUSSION TITLE
How to manage the version of documentation, not Gatsby

### DISCUSSION THREAD

**Discussion by xunyao4dev** [gatsbyjs/gatsby#34264] [Help]
How to manage the version of documentation, just like this
![image](https://user-images.githubusercontent.com/87527853/146364173-67a13f98-af49-4356-bfd2-e6ed0d66737e.png)

**Comment by violy** [comment]
![image](https://user-images.githubusercontent.com/881828/146434443-4056e4c1-d48c-45b4-981d-b26e67151fc5.jpeg)

in the Gatsby documentation main dropdown, you can switch between major versions.
